craco.config.cjs

```diff
@@ -93,6 +94,12 @@ module.exports = {
       // See https://vanilla-extract.style/documentation/integrations/webpack/#identifiers for docs.
       // See https://github.com/vanilla-extract-css/vanilla-extract/issues/771#issuecomment-1249524366.
       new VanillaExtractPlugin({ identifiers: 'short' }),
+      new RetryChunkLoadPlugin({
+        retryDelay: `function(retryAttempt) {
+          return 2 ** (retryAttempt - 1) * 1000;
```

Comment:
This first retry will not occur for 1 second. Is this the intended behavior?
I think this is the same as the old behavior - just double-checking.

Reply:
yeah this is same as existing behavior, but i'm open to changing this around if we want. could do 500ms to start?